Tasks/Gradle/gradletask.ts

```diff
@@ -94,43 +96,52 @@ if (specifiedJavaHome) {
     tl.debug('Set JAVA_HOME to ' + specifiedJavaHome);
     process.env['JAVA_HOME'] = specifiedJavaHome;
 }
+ 
+/* Trigger code coverage enable flow and then build */
+enableCodeCoverage()
+    .then(function (resp) {
+        tl.debug("Enabled code coverage successfully: " + reportingTaskName);
+        gb.arg(reportingTaskName);
+    }).catch(function (err) {
+        tl.warning("Failed to enable code coverage: " + err);
+    }).fin(function () {
+        setGradleOpts();
```

Comment:
+1

---
In reply to: [77330590](https://github.com/Microsoft/vsts-tasks/pull/2402#discussion_r77330590) [](ancestors = 77330590)